packages/core/database/src/__tests__/index.test.ts

```diff
@@ -1,4 +1,4 @@
-import { Database, DatabaseConfig } from '../index';
+import {Database, DatabaseConfig} from '../index';
```

Comment:
```suggestion
import { Database, DatabaseConfig } from '../index';
```
I think you have different linting rules set up than the onse used in the strapi monorepo. Imports should be formatted like this everywhere.

Reply:
Fixed.